src/components/playground/ActionsBar.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+import MdThumbUp from 'react-icons/lib/md/thumb-up';
+import FaSmileO from 'react-icons/lib/fa/smile-o';
+import FaEllipsisH from 'react-icons/lib/fa/ellipsis-h';
+
+import { likeComment, unLikeComment } from '../../actions/playground';
+
+@connect(state => state.playground)
+@Radium
+class ActionsBar extends React.Component {
+
+  likeClickHandler(index) {
+    if (!this.props.liked) {
+      this.props.dispatch(likeComment(index));
+    } else {
+      this.props.dispatch(unLikeComment(index));
+    }
+  }
+
+  moreClickHandler() {
+    if (this.props.moreClickHandler) { this.props.moreClickHandler(); }
+  }
+
+  reactionClickHandler() {
+    
+  }
+
+  render() {
+
+    return (
+      <div style={ styles.statsBar }>
+        { 
+          this.props.togglerGroups['interaction'].togglers['likes'].status ? 
```

Comment:
No reason, I should go with dot notation I guess, all the keys in those objects are clear for dot notation.